Tests/Marketplace/marketplace_services.py

```diff
@@ -2346,7 +2346,20 @@ def _collect_pack_tags(self, user_metadata, landing_page_sections, trending_pack
 
         return tags
 
-    def _enhance_pack_attributes(self, index_folder_path, dependencies_metadata_dict,
+    def _get_tags_by_marketplace(self, user_metadata: dict, marketplace: str):
+        """ Returns tags in according to the current marketplace"""
+        tags = []
+        for tag in user_metadata.get('tags', []):
+            if ':' in tag:
+                tag_data = tag.split(':')
+                if marketplace in tag_data[0]:
```

Comment:
Need to split to multiple marketplaces, with a comma as a separator.

Reply:
Actually if we got to this point, the marketplaces are valid (There will be a validation in the demisto-sdk for it)

Reply:
I agree with Tal here, it's a small change to add support for this

Reply:
@dantavori Talked with Tal about it, tag_data[0] is a string that contains all of the marketplaces that are valid for the current tag, so if `marketplace == "xsoar"` and `tag_data[0] == "xsoar,marketplacev2"`, we will include the tag in the current pack upload